### ISSUE TITLE
Bug: One declaration per line has issue with named parameters

### ISSUE THREAD

**Issue by bgriffith** [sasstools/sass-lint#51]
Named parameters cause issue with one declaration per line. Best example for these are the [Other color functions](http://sass-lang.com/documentation/Sass/Script/Functions.html#other_color_functions) with their optional parameters

The following tests currently fail:
```scss 

// The following all cause issues
$foo: adjust-color(rgb(255, 0, 0), $lightness: -20%);

.bar {
  color: adjust-color(#102030, $blue: 5);
  background-color: scale-color(hsl(120, 70%, 80%), $lightness: 50%);
  border-color: change-color(hsl(25, 100%, 80%), $lightness: 40%, $alpha: 0.8);
}
```

Using v1.0.0

**Comment by Snugug** [comment]
If you keep finding and closing these issues, I'm may just make you a collaborator (if you're interested)

**Comment by bgriffith** [comment]
Sure - sounds good but just glad to help! Working on a fix for this - should be sorted soon. Just ignoring if parent is an argument.

**Comment by Snugug** [comment]
Yup, that's what I'm thinking too.

**Comment by Snugug** [comment]
@bgriffith You've been added as a collaborator! Please continue to issue PRs to update code, but other than that you've just gotta accept the invite!

**Comment by bgriffith** [comment]
Awesome thanks! Will do. PR up.
